frontend/src/Components/LoginForm.jsx

```
@@ -118,24 +128,21 @@ function LoginForm() {
-          <div className="divider">
-            <span>or</span>
-          </div>
-
-          <button type="button" className="google-btn">
-            <img src={"images/google.png"} alt="Google" />
-            Login with Google
+          
+          <button type="submit" disabled={isLoading}>
+            {isLoading ? (
+              <>
+                <i className="fas fa-spinner fa-spin"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)